tests/e2e/whitelisted-provider-1/yagna_rCURRENT.log

```diff
@@ -0,0 +1,77 @@
+[2023-10-04T09:01:47.313+0000 INFO  yagna] Starting yagna service! Version: 0.12.2 (8efd8657 2023-06-06 build #296).
```

Comment:
@cryptobench How could you :D

Reply:
The reason for these two datadirs is the fact that one of the examples requires an array of whitelisted provider ID's. The default way I spawn them, I have no idea what their node ID's will be, thus I needed two "hardcoded" providers available. 

By having the datadir, i'm ensuring the same Provider ID is being used. I am not sure what the bare-minimum requirements are for storing the files that configures this within yagna, thus the reason for all the databases etc..